pylmod/client.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Client classes and methods for PyLmod module
+"""
+
+import logging
+
+log = logging.getLogger('pylmod.client')  # pylint: disable=invalid-name
+
+class Client(object):
+
+    def get_academicterms(self):
+        pass
```

Comment:
These should all raise `NotImplementedError` exceptions instead of just silently doing nothing